src/bin/server.rs

```diff
@@ -138,6 +139,47 @@ fn feed(page: i64) -> Json<Value> {
     }))
 }
 
+#[get("/link/<link>/comments")]
```

Comment:
Just about the naming, I'd like to call it `link_id` instead of `link`

Reply:
You're right.

I'm used to write controllers taking not the id of a model but the model itself (thing we may be able to do with a rocket guard?). The cool thing about that is you don't need to query the model and that if the id doesn't match any row in database it automatically returns a 404.

Reply:
Ah, `link_id` conflicts with the name of the field imported into scope by `use codedaily_backend::schema::comments::dsl::*;`. So we have to either rename the parameter or the alias the imported field. Which do you prefer?